manual/manual/tutorials/objectexamples.etex

```diff
@@ -1091,11 +1091,13 @@ class virtual comparable =
   end;;
 \end{caml_example}
 We then define a subclass "money" of "comparable". The class "money"
-simply wraps floats as comparable objects. We will extend it below with
-more operations. We have to use a type constraint on the class parameter "x"
-because the primitive "<=" is a polymorphic function in
-OCaml.  The "inherit" clause ensures that the type of objects
-of this class is an instance of "#comparable".
+simply wraps floats as comparable objects.\footnote{floats are an
+approximation of decimal numbers, they are unsuitable for use in most
+monetary calculations as they may introduce errors.}  We will extend
```

Comment:
A last minor nitpick: I would rather say `floats can only approximate decimal numbers` since `decimal numbers` is not the only set of numbers that floats are trying to approximate. Maybe adding a modal in the form of `most serious monetary calculation` would be better too but I will let you decide.

Reply:
gentle ping since the deadline for documentation fixes for 4.12.0 is approaching (you could also add a change entry if you wish so)?